app/controllers/available_hours_controller.rb

```diff
@@ -0,0 +1,7 @@
+class AvailableHoursController < ApplicationController
+  def index
+    unavailable_hours = AvailableHour.new(params[:date])
+                                     .get_unavailable_hours
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.